packages/fcalclusterer/package.py

```diff
@@ -29,6 +29,7 @@ class Fcalclusterer(CMakePackage):
 
     # CMAKE_INSTALL_PREFIX is overwritten by the package
     patch("install.patch", when="@:1.0.1")
+    patch("random-shuffle-c17.patch")
```

Comment:
From the issue you opened, I gather this has been fixed in master. In that case I'd suggest to do:
```suggestion
    patch("random-shuffle-c17.patch", when="@:1.0.1")
```